### PR TITLE
Intention from doc

### PR DESCRIPTION
**Pull request by keerath**

### CODE REVIEW

File: core/src/main/scala/com/kodebeagle/ml/LDAModel.scala [hunk 1]
```diff
@@ -88,7 +88,7 @@ class DistributedLDAModel private[ml] (
   /**
    * Compute the log likelihood of the observed tokens in the training set,
    * given the current parameter estimates:
-   *  log P(docs | topics, topic distributions for docs, alpha, eta)
+   *  log P(propDocs | topics, topic distributions for propDocs, alpha, eta)
```
Comment (sachintyagi22) [inline core/src/main/scala/com/kodebeagle/ml/LDAModel.scala]: Why this?

File: core/src/main/scala/com/kodebeagle/spark/CreateTopicModelJob.scala [hunk 2]
```diff
@@ -312,7 +312,7 @@ object CreateTopicModelJob extends Logger {
                      fileIdVsName: mutable.Map[Long, String]): Unit = {
     repoSummary.groupBy(_._1).foreach(f => {
       if (repoIdVsName.get(f._1).getOrElse("") != "") {
-        log.info(s"Top docs for repo : ${repoIdVsName.get(f._1).getOrElse("Anonymous")}")
+        log.info(s"Top propDocs for repo : ${repoIdVsName.get(f._1).getOrElse("Anonymous")}")
```
Comment (sachintyagi22) [inline core/src/main/scala/com/kodebeagle/spark/CreateTopicModelJob.scala]: ?

File: core/src/main/scala/com/kodebeagle/util/SparkIndexJobHelper.scala [hunk 3]
```diff
@@ -43,6 +43,8 @@ object SparkIndexJobHelper {
 
   def createSparkContext(conf: SparkConf): SparkContext = {
     val sc = new SparkContext(conf)
+    conf.setMaster(KodeBeagleConfig.sparkMaster)
+    conf
```
Comment (sachintyagi22) [inline core/src/main/scala/com/kodebeagle/util/SparkIndexJobHelper.scala]: Why this dangling conf?

File: core/src/main/scala/com/kodebeagle/spark/TypeAggregationJob.scala [hunk 4]
```diff
@@ -44,7 +46,47 @@ object TypeAggregationJob extends Logger {
       args(0).trim
     }.toOption
 
-    sc.textFile(typesInfoLocation.getOrElse(KodeBeagleConfig.typesInfoLocation))
+    val partitions = Try {
+      args(1).trim.toInt
+    }.toOption.getOrElse(500)
+
+    val typesInFile: RDD[TypesInFile] = readTypesInFile(sc, typesInfoLocation)
```
Comment (sachintyagi22) [inline core/src/main/scala/com/kodebeagle/spark/TypeAggregationJob.scala]: Can you write comments on changes made here? Not immediately clear what these are about.

**Comment by keerath** [comment]
@sachintyagi22 Updated this pr which the changes mentioned above

**Pull request closed without merging**